src/components/fieldset/fieldset.html

```diff
@@ -1,6 +1,38 @@
-<section>
-  <fieldset></fieldset>
-</section>
+<form>
+  <section>
+    <p>
+      <label>
+        <span>Enter Task</span>
+        <abbr title="required">*</abbr>
+      </label>
+      <input type="text" id="task" name="task" />
+    </p>
+
+    <fieldset>
+      <legend>WHEN</legend>
+      <ul>
+        <li>
```

Comment:
No `ul` and `li` needed, the form and its structure already form a semantic collection of elements